consensus/Cargo.toml

```diff
@@ -23,7 +23,7 @@ workspace = true
 async-trait = "0.1"
 futures = { package = "futures-util", version = "0.3" }
 futures-executor = { version = "0.3" }
-instant = { version = "0.1", features = [ "wasm-bindgen" ] }
+instant = { version = "0.1", features = ["wasm-bindgen"] }
```

Comment:
I think we would want to introduce a feature called `wasm_bindgen` that enables this other feature such that it doesn't always get pulled. Something like:
```
[features]
wasm-bindgen = ["instant/wasm-bindgen"]